### PR TITLE
Add support for Rmilter/Rspamd

### PR DESCRIPTION
**Pull request by mundschenk-at**
* detects both spam and viruses
* no need for configuration

### CODE REVIEW

File: modoboa_stats/management/commands/logparser.py [hunk 1]
```diff
@@ -322,13 +339,59 @@ def _parse_amavis(self, log, host, pid):
...
-                self.inc_counter(m.group(2), "virus")
-            elif m.group(1) in ["SPAM", "SPAMMY"]:
-                self.inc_counter(m.group(2), "spam")
+        m = self._regex["amavis"].search(log)
+        if m is not None:
+            dom = m.group("domain")
+            spam_result = m.group("result")
+            if dom is not None and dom in self.domains:
+                if spam_result == "INFECTED":
+                    self.inc_counter(dom, 'virus')
```
Comment (tonioo) [inline modoboa_stats/management/commands/logparser.py]: double quotes please ;)

File: modoboa_stats/management/commands/logparser.py [hunk 2]
```diff
@@ -322,13 +339,59 @@ def _parse_amavis(self, log, host, pid):
...
+                elif spam_result in ["SPAM", "SPAMMY"]:
+                    self.inc_counter(dom, 'spam')
```
Comment (tonioo) [inline modoboa_stats/management/commands/logparser.py]: Same here.

File: modoboa_stats/management/commands/logparser.py (hunk 1 above)
Comment (mundschenk-at) [inline modoboa_stats/management/commands/logparser.py]: Sure. Sorry, don't know how those got mixed in again.

**Comment by mundschenk-at** [comment]
@tonioo single quotes fixed.

**Pull request merged**